src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -36,12 +36,31 @@
 
     private static final long serialVersionUID = 1;
 
-    public PullRequestSCMHead(int number) {
+    private String user;
```

Comment:
@imeredith I would use `login` because it is the term used by GitHub on his API documentation.